docs/guides/request_loaders.mdx

```diff
@@ -0,0 +1,126 @@
+---
+id: request-loaders
+title: Request loaders
+description: How to manage the requests your crawler will go through.
+---
+
+import ApiLink from '@site/src/components/ApiLink';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+import RlBasicExample from '!!raw-loader!./code/request_loaders/rl_basic_example.py';
+
+import TandemExample from '!!raw-loader!./code/request_loaders/tandem_example.py';
+import ExplicitTandemExample from '!!raw-loader!./code/request_loaders/tandem_example_explicit.py';
+
+The [`request_loaders`](https://github.com/apify/crawlee-python/tree/master/src/crawlee/request_loaders) sub-package extends the functionality of the <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>, providing additional tools for managing URLs. If you are new to Crawlee, and you do not know the `RequestQueue`, consider starting with the [Storages](https://crawlee.dev/python/docs/guides/storages) guide first. Request loaders define how requests are fetched and stored, enabling various use cases, such as reading URLs from files, external APIs or combining multiple sources together.
+
+## Overview
+
+The [`request_loaders`](https://github.com/apify/crawlee-python/tree/master/src/crawlee/request_loaders) sub-package introduces the following abstract classes:
+
+- <ApiLink to="class/RequestLoader">`RequestLoader`</ApiLink>: The base interface for reading requests in a crawl.
+- <ApiLink to="class/RequestManager">`RequestManager`</ApiLink>: Extends `RequestLoader` with write capabilities.
+- <ApiLink to="class/RequestManagerTandem">`RequestManagerTandem`</ApiLink>: Combines a read-only `RequestLoader` with a writable `RequestManager`.
+
+And one specific request loader:
+
+- <ApiLink to="class/RequestList">`RequestList`</ApiLink>: A lightweight implementation of request loader for managing a static list of URLs.
+
+Below is a class diagram that illustrates the relationships between these components and the <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>:
+
+```mermaid
+classDiagram
+
+%% ========================
+%% Abstract classes
+%% ========================
+
+class BaseStorage {
+    <<abstract>>
+    _attributes_
+    _methods_()
+}
+
+class RequestLoader {
+    <<abstract>>
+    _attributes_
+    _methods_()
+}
+
+class RequestManager {
+    <<abstract>>
+    _attributes_
+    _methods_()
```

Comment:
Would be nice to list the most important methods here so that it's evident that Loader is read-only and manager is read-write.

Reply:
![image](https://github.com/user-attachments/assets/a1a2c4b5-9a7d-4c8f-9fb1-cd2142be0746)